config/initializers/config_check.rb

```diff
@@ -3,9 +3,7 @@
 # Check key settings in production to make sure they have safe and sensible values.
 
 # rubocop:disable Style/IfUnlessModifier
-# Don't abort if we're running via a rake task b/c we need to be able to run migrations on deploy
-# since migrations will copy the config to .env.
-if Rails.env.production? && File.split($PROGRAM_NAME).last != "rake"
+if Rails.env.production?
```

Comment:
Don't need to check for rake anymore b/c we assume the migration is already complete by this point. If it isn't then a bunch of other things will break anyway.